### DISCUSSION TITLE
Add setting for making the default ctrl + enter configurable

### DISCUSSION THREAD

**Discussion by woidda** [community/community#11228] [Pull Requests]
We should have an option in [settings/accessibility](https://github.com/settings/accessibility) to change the behavior of the `[ctrl] + [enter]` keyboard short cut. "Start a review" as default is not the best default for everyone. For me it is very annoying, because sometimes you discuss a comment and need to add extra explanation to it. Additionally, every else at github the `[ctrl] + [enter]` does a submit with an immediate action and not "put it on some pile for later" 

See also https://github.com/refined-github/refined-github/issues/312 

![image](https://user-images.githubusercontent.com/459937/153159471-1a122030-b607-47ae-ab46-fe6864fd265f.png)

**Comment by StingyJack** [comment]
So many times this has caused me to create a review when I just wanted to add a comment.
